GANDLF/config_manager.py

```diff
@@ -738,4 +738,10 @@ def ConfigManager(
     Returns:
         dict: The parameter dictionary.
     """
-    return _parseConfig(config_file_path, version_check_flag)
+    try:
+        return _parseConfig(config_file_path, version_check_flag)
+    except Exception as e:
+        logging.info(
```

Comment:
Minor thing, but perhaps we should use logging.error here to indicate errors? The log is easier to read

Reply:
Done.